Userland/Libraries/LibGfx/ICC/Profile.h

```diff
@@ -211,12 +211,7 @@ struct ProfileHeader {
 class Profile : public RefCounted<Profile> {
 public:
     static ErrorOr<NonnullRefPtr<Profile>> try_load_from_externally_owned_memory(ReadonlyBytes);
-
-    Profile(ProfileHeader const& header, OrderedHashMap<TagSignature, NonnullRefPtr<TagData>> tag_table)
-        : m_header(header)
-        , m_tag_table(move(tag_table))
-    {
-    }
+    static ErrorOr<NonnullRefPtr<Profile>> try_create(ProfileHeader const& header, OrderedHashMap<TagSignature, NonnullRefPtr<TagData>> tag_table);
```

Comment:
The `try_` prefix is exclusively reserved for when there's already an infallible function of the same name. `TRY(create())` > `TRY(try_create())` :)

(someone should've rejected `try_load_from_externally_owned_memory` above, too :smile:)

Reply:
(`try_load_from_externally_owned_memory` probably predates this rule, iirc it's recent?)

Reply:
Not that recent, basically since started having these conflicts. As usual, it's probably not written down anywhere though :stuck_out_tongue: